bin/ci_check

```diff
@@ -56,7 +56,7 @@ if (sys.version_info < (3,0,0)):
 ################################################################################
 # Globals....
 name_of_ci_check_regression = '<core>_ci_check_dev' # must match the name of one regression list in .metrics.json
-core_tests                  = ['misalign', 'illegal', 'dhrystone', 'fibonacci', 'riscv_ebreak_test_0']
+core_tests                  = ['misalign', 'illegal', 'dhrystone', 'fibonacci'] #, 'riscv_ebreak_test_0'] # FIXME: full pathname of this test is too long for Verilator (seriously!)
```

Comment:
This will only affect ci_check for the "core" testbench using verilator, is that right? It does not affect ci_check for uvmt?

Out of curiousity, what is the problem with the full pathname? 'riscv_ebreak_test_0' doesn't seem much longer than the other items in that list.

Reply:
That is correct - this change only affects the core testbench running under Verilator.  (The regression list for the the UVM ci_check comes from the .metrics.json file.)

The issue with riscv_ebreak_test_0 is not necessarily it's filename, it is the full pathname.  I have a _lot_ of working copies of repos (not just OpenHW ones) and keep them organized with a rather tall directory tree.  I've poked around looking for a solution for this that doesn't require a re-compile of Verilator using custom macros without success.  The "best' workaround is to update the Makefile to generate a relative path for Verilator - I've pushed in an update for that to this PR.